src/wallet/wallet.cpp

```diff
@@ -2432,6 +2432,7 @@ void CWallet::AvailableCoins(interfaces::Chain::Lock& locked_chain, std::vector<
     // Either the WALLET_FLAG_AVOID_REUSE flag is not set (in which case we always allow), or we default to avoiding, and only in the case where
     // a coin control object is provided, and has the avoid address reuse flag set to false, do we allow already used addresses
     bool allow_used_addresses = !IsWalletFlagSet(WALLET_FLAG_AVOID_REUSE) || (coinControl && !coinControl->m_avoid_address_reuse);
+    const int min_depth = { coinControl ? coinControl->m_min_depth : 0 };
```

Comment:
nit, could drop `{ }`?

Reply:
I'd rather drop the `=`. Note that the `{}` enable the `-Wnarrowing` compiler check

Reply:
Or keep it as is, since the compiler checks seems to work even with the `=`

Reply:
hm. In a simple example c++ program, I get a compiler error when I drop the `=`. so I added it.

Reply:
Huh? Both pass for me

```cpp
$ echo 'int main(){ const int min_depth = { true ? 1 : 0 };}' | g++ -x c++ -
$ echo 'int main(){ const int min_depth   { true ? 1 : 0 };}' | g++ -x c++ -

Reply:
FYI, even without optimizations these two are equivalent https://godbolt.org/z/Q4sjcx